docs/streams.html

```diff
@@ -279,8 +279,8 @@
 <pre>
     KStreamBuilder builder = new KStreamBuilder();
 
-    KStream<String, GenericRecord> source1 = builder.stream("topic1", "topic2");
-    KTable<String, GenericRecord> source2 = builder.table("topic3", "stateStoreName");
+    KStream&lt;String, GenericRecord&gt; source1 = builder.stream("topic1", "topic2");
```

Comment:
@mjsax At least my browser (Safari on Mac) doesn't show the types between `<` and `>` ([here](http://kafka.apache.org/documentation#streams_dsl_source)). It would be great if you can check and confirm whether it shows fine on your side. Thanks.